schema/vrs.yaml

```diff
@@ -160,12 +162,20 @@ definitions:
   # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
   # SystemicVariation
 
-  AbsoluteAbundance:
+  Abundance:
+    additionalProperties: false
+    description: >-
+      The measure of a quantity of a molecule in a system.
+    oneOf:
+      - $ref: "#/definitions/CopyNumber"
+    discriminator:
+      propertyName: type
+
+  CopyNumber:
     additionalProperties: false
     type: object
     description: >-
-      Absolute abundance of a systemic subject such as a gene or
-      specific type of molecular variation.
+      The count of copies of a Molecular Feature or Molecular Variation subject within a genome.
```

Comment:
Once we're allowing MF and MV, why not allow Sequence Locations too?

Reply:
I agree. Seems like a natural extension. We've gone back and forth on this a few times. I suppose we can allow it and deprecate down the road if VR group agrees and we later realize issues with it.